crates/corro-types/Cargo.toml

```diff
@@ -49,6 +49,7 @@ tracing = { workspace = true }
 tripwire = { version = "0.1.0-alpha.0", path = "../tripwire" }
 uhlc = { workspace = true }
 uuid = { workspace = true }
+cuckoofilter = { version = "0.5.0" }
```

Comment:
Is this still needed, considering the code using it was commented out?

Reply:
Ah yeah, I started on something and didn't follow through. I'll remove!